src/Wallabag/CoreBundle/Resources/views/base.html.twig

```diff
@@ -76,7 +76,7 @@
         {% block footer %}{% endblock %}
 
         {% if craue_setting('piwik_enabled') %}
-            {{ piwik(craue_setting('piwik_host'), craue_setting('piwik_site_id')) }}
+            {{ matomo(craue_setting('piwik_host'), craue_setting('piwik_site_id')) }}
```

Comment:
I was wondering if we should change them too: `piwik_host`, `piwik_site_id` & `piwik_enabled ` ?

Reply:
It's only because we need a migration ... that's why I didn't write it.